File: CPP/Swap.cpp
#include<iostream>
using namespace std;
int main(int argc, char const *argv[])
{
    int a=10, b=20;
    cout<<"Before swapping: a = "<<a<<" and b = "<<b<<endl;
    int c = a;
    a = b;
    b = c;
    cout<<"After swapping: a = "<<a<<" and b = "<<b<<endl;
    return 0;
}

File: Java/Swap.java
public class Swap {
    public static void main(String[] args) {
        int a=10, b=20;
        System.out.println("Before swapping: a = " + a + " and b = " + b);
        int c = a;
        a = b;
        b = c;
        System.out.println("After swapping: a = " + a + " and b = " + b);
    }
}